src/System.Reflection/tests/project.json

```diff
@@ -1,13 +1,16 @@
 {
   "dependencies": {
+    "System.AppContext": "4.0.0",
     "System.Collections": "4.0.10",
     "System.Diagnostics.Debug": "4.0.10",
     "System.Globalization": "4.0.10",
     "System.IO": "4.0.10",
+    "System.IO.FileSystem": "4.0.0.0",
     "System.Linq": "4.0.0",
-    "System.Reflection": "4.0.10",
+    "System.Reflection.Emit": "4.0.0",
```

Comment:
All I want to be careful about is that when we start running these test on .NET Native that we don't loose all the reflection tests because of these new tests. If we think that might happen perhaps we should factor some of these out into another test project.